src/Backend/Modules/ContentBlocks/Actions/ContentBlockAdd.php

```diff
@@ -12,26 +12,22 @@
 use Backend\Core\Engine\Authentication;
 use Backend\Core\Engine\Base\ActionAdd as BackendBaseActionAdd;
 use Backend\Core\Engine\Model as BackendModel;
-use Backend\Modules\ContentBlocks\Command\CreateContentBlock;
-use Backend\Modules\ContentBlocks\Event\ContentBlockCreated;
-use Backend\Modules\ContentBlocks\Form\ContentBlockType;
+use Backend\Modules\ContentBlocks\Domain\ContentBlock\Command\CreateContentBlock;
+use Backend\Modules\ContentBlocks\Domain\ContentBlock\Event\ContentBlockCreated;
+use Backend\Modules\ContentBlocks\Domain\ContentBlock\ContentBlockType;
+use Symfony\Component\Form\Form;
 
 /**
  * This is the add-action, it will display a form to create a new item
  */
-class Add extends BackendBaseActionAdd
+class ContentBlockAdd extends BackendBaseActionAdd
```

Comment:
I think in this case renaming this might be a bit overkill, but if you do it you need to make sure that all the changes needed to the database are added to the upgrade guide

Goes for all the other actions in this case as well.